corehq/apps/notifications/migrations/0008_auto_20220329_2130.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.27 on 2022-03-29 21:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
nit: I'm not sure if it is an official convention or not, but it is better to manually rename the migration to something that better conveys what the migration was for.

Reply:
Noted! However I'm going to make the migration its own PR.

Reply:
Even better 👍🏻